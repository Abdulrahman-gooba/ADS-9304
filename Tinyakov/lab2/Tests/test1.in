(cos(+(a)(*(c)(d)))) 4
