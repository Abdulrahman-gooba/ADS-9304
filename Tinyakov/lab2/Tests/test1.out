Wrong variable for derived: '4'
